tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator.v0.0.9.yml

```diff
@@ -0,0 +1,14 @@
+name: mock-operator.v0.0.9
```

Comment:
```suggestion
name: mock-operator-invalid-env-value.v0.0.9
```

Reply:
And we could maybe rename `tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator.v0.0.9.yml` to `tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator-invalid-env-value.v0.0.9.yml`

Reply:
Or if that is not good, we could at least put a comment in these files to tell what the point of them are. It is a little confusing just having multiple versions of the addon in the test data folder without any description of what they are testing